### ISSUE TITLE
blob-scheme block causes riot.im to logout

### ISSUE THREAD

**Issue by tleydxdy** [uBlockOrigin/uMatrix-issues#126]
### Prerequisites

- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
    - [x] I also searched the existing issues at <https://github.com/gorhill/uMatrix/issues>
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uMatrix](https://old.reddit.com/r/uMatrix/)
- I tried to reproduce the issue when...
    - [ ] uMatrix is the only extension
    - [ ] uMatrix with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uMatrix
- [x] I checked the [documentation](https://github.com/gorhill/uMatrix/wiki) to understand that the issue I report is not a normal behavior
- [x] I used the logger to rule out that the issue is caused by my ruleset

### Description

blob:https://riot.im are blocked from accessing https://riot.im's localstorage, causes riot.im to loggout (e2ee room will breake) after enabling "all" for blob-scheme things worked 

### A specific URL where the issue occurs

https://riot.im/app

### Steps to Reproduce

1. view image in a e2ee room (led to blob:https://riot.im/blablabla
2. go back
3. logged out

### Ruleset

riot.im allow all, blob-scheme default

### Supporting evidence

logger shows a localstorage access is denied

### Your environment

* uMatrix version: 1.3.16
* Browser Name and version: Firefox 66.0.2
* Operating System and version: Arch

**Comment by uBlock-user** [comment]
https://github.com/gorhill/uMatrix#regarding-broken-sites

**Comment by tleydxdy** [comment]
I think that blob:https://riot.im should be considered as https://riot.im by umatrix, I see no reason why it shouldn't be.

**Comment by gorhill** [comment]
> I think that  blob:https://riot.im` should be considered as `https://riot.im` by umatrix

That sounds reasonable, i will consider.

**Comment by tleydxdy** [comment]
by the same token, stuff like view-source: and the like should be treated the same as well

**Comment by uBlock-user** [comment]
> view-source:

view-source: is a browser scheme, so it won't be the same.

**Comment by gorhill** [comment]
Right, `view-source` is the browser making the request to pull data, while `blob` it can be argued that is the site's javascript which is trying to wrap data as a URL.
